src/components/button/button.scss

```diff
@@ -5,11 +5,14 @@
 @import "./../../style-config/buttons";
 
 .carbon-button {
+  align-items: center;
   border: $carbon-button__border--secondary;
   border-radius: $carbon-button__border-radius;
   box-sizing: border-box;
   cursor: pointer;
-  display: inline-block;
+  display: inline-flex;
```

Comment:
Looks fine in Chrome & Firefox.

Menu button broken in Edge but regular button looks fine.

Don't have IE11 to test locally.